packages/dfw/src/dfwcomponents/WorkOrderCell/WorkOrderCellNavigation.tsx

```diff
@@ -19,6 +19,7 @@ type WorkOrderCellNavigationProps = WorkOrderCellProps & {
     onPress: () => void;
     leftSwipeGroup?: SwipeGroup;
     rightSwipeGroup?: SwipeGroup;
+    bookmarked?: boolean;
```

Comment:
What do you think about naming this `isBookmarked` instead? Very minor, but I think (?) other boolean props we use are named in the same way.

Reply:
Ye, i agree. i renamed stuff to isBookmarked instead now.